cbmackintosh/fast-command-engine
# Feature/user authentication

**Clearly and concisely describe the feature.**
- Sets up the rails application with a basic User model with all gems, controllers etc needed for authentication

**Is this a feature or refactor?**
- Feature

**Problems Encountered & Resulting Solutions**
- Unsure how to update the model with new User attributes moving forward since this application involves PostgreSQL
- Unsure how to deploy to heroku as well as integrate Travis for continuous integration and automatic deployment

**How should this be tested?**
- Run locally with the FE application and make sure the rails application responds to authentication requests correctly

**Steps To Take**
- Update User model
- Add models for contacts, incidents, incident posts etc
